src/streamlink_cli/main.py

```diff
@@ -984,6 +988,26 @@ def check_root():
             console.logger.info("streamlink is running as root! Be careful!")
 
 
+def check_current_versions():
```

Comment:
maybe this method should have a different name, like log current versions? might be confusing otherwise...